src/components/Captions/imagecaptions.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+const CaptionContainer = styled.div`
+  display: flex;
+  flex-flow: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 480px;
+  text-align: left;
+`;
+
+const CaptionTitle = styled.h4`
+  font-family: "Raleway";
```

Comment:
Is this saved to a CSS variable you can use?

Reply:
@erinkelley27 waiting on his to be made, the variable, just waiting on it.